configuration.nix

```diff
@@ -0,0 +1,147 @@
+# Edit this configuration file to define what should be installed on
+# your system.  Help is available in the configuration.nix(5) man page
+# and in the NixOS manual (accessible by running ‘nixos-help’).
+
+{ config, pkgs, ... }:
+
+{
+  imports =
+    [ # Include the results of the hardware scan.
+      ./hardware-configuration.nix
+      ./modules
```

Comment:
Do you expect people to use this by cloning the repo and then modifying the configuration.nix locally? Because that's not really the intended approach.